src/Storage/Commands.Storage/Common/Cmdlet/DisableAzureStorageStaticWebsite.cs

```diff
@@ -23,6 +23,8 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet
     /// <summary>
     /// Disable azure storage service StaticWebsite, currently only available on Blob service
     /// </summary>
+    [Cmdlet("Disable", Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "StorageStaticWebsite", SupportsShouldProcess = true), 
```

Comment:
@cormacpayne 
Actually, I use "VerbLifecycle.Disable" before, but I see all other cmdlets use string instead of Powershell Verb in the cmdlets name. So I change it to string. Then what is the recommend cmdlet name? Why we change it from verb to string in the before change? 
@maddieclayton , @MiYanni , do you have any idea for this?

Following are 2 samples of current code:

https://github.com/Azure/azure-powershell/blob/899f2290616bbd67fbd79d5f1abd4ba1d4351bb4/src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/DisableAzureDiskEncryption.cs#L29


https://github.com/Azure/azure-powershell/blob/preview/src/Storage/Commands.Storage/Common/Cmdlet/EnableAzureStorageDeleteRetentionPolicy.cs#L26

Reply:
@blueww interesting, I must have missed this change. I'll sync up with @maddieclayton offline and see why this change was made. How you have the verb defined in your cmdlets now is OK for this PR to go in.